lm-agent/Makefile

```diff
@@ -19,18 +19,18 @@ mypy: install
 
 .PHONY: lint
 lint: install
-	poetry run ruff check ${PACKAGE_NAME}
-	poetry run ruff check tests
+	poetry run ruff check tests ${PACKAGE_NAME}
```

Comment:
It is a bit unclear to me that both `make lint` and `make format` run both format and lint verifications. I see arguments make one of them just verify, and the other try to fix, meybe we just need to clarify the names.

Reply:
I’m ok with validate and modify